apps/router-e2e/__e2e__/native-tabs/app/_layout.tsx

```diff
@@ -0,0 +1,59 @@
+import { ThemeProvider, DarkTheme } from '@react-navigation/native';
+import { Badge, Icon, Title } from 'expo-router';
+import { NativeTabs } from 'expo-router/build/ui/NativeBottomTabs/NativeBottomTabsNavigator';
+import { AndroidIcon, IOSIcon } from 'expo-router/build/ui/NativeBottomTabs/NavigatorElements';
+import { Appearance, Platform } from 'react-native';
+
+Appearance.setColorScheme('dark');
+
+export default function Layout() {
+  return (
+    <ThemeProvider value={DarkTheme}>
+      <NativeTabs
+        style={{
+          // fontSize: 16,
+          // fontWeight: 700,
+          // fontStyle: 'italic',
+          // fontFamily: 'Courier New',
+          backgroundColor: Platform.OS === 'android' ? 'black' : undefined,
+          // badgeBackgroundColor: 'green',
+          // color: 'blue',
+          tintColor: 'orange',
+          blurEffect: 'systemChromeMaterial',
+          labelVisibilityMode: 'auto',
+          rippleColor: 'orange',
+          iconColor: Platform.OS === 'android' ? '#888' : undefined,
+          color: Platform.OS === 'android' ? '#888' : undefined,
+          '&:active': {
+            fontSize: 14,
+            indicatorColor: 'black',
+          },
+        }}
+        minimizeBehavior="onScrollDown">
+        <NativeTabs.Trigger
+          name="index"
+          options={{
+            icon: { sfSymbolName: 'applewatch.side.right' },
+            iconResourceName: 'ic_phone',
+            title: 'My Watch',
+          }}
+        />
+        <NativeTabs.Trigger name="faces" options={{ title: 'Face Gallery' }} popToRoot>
+          <IOSIcon name="lock.applewatch" />
+          <IOSIcon name="lock.open.applewatch" useAsSelected />
+          <AndroidIcon name="ic_lock_open" />
+          <Title>Face Gallery</Title>
```

Comment:
Would also nice to have a similar API for link previews so it's not options in one API and components in another. Ideally the exact same components would be used across both.

Reply:
Let's do it like this, but I think we can change this to `native-tabs` in beta if everything seems good

Unstable can discourage people from using the tabs